package.json

```diff
@@ -11,7 +11,6 @@
     "serve": "docusaurus serve"
   },
   "dependencies": {
-    "@cmfcmf/docusaurus-search-local": "latest",
```

Comment:
should it be commented-out or removed?

Reply:
JSON doesn't like comments.
![image](https://user-images.githubusercontent.com/67106617/119464546-1e13ba80-bd43-11eb-9312-a94dce384e60.png)

Reply:
ah, right, so it should be described only in docs what should be done to enable it